PAPER/FIT_Paper_Ex1/Part_1.tex

```diff
@@ -1,52 +1,56 @@
-those tools may be used for Research and Development training as they give an
-access to key point data related to the fuel cycle. 
+Fuel cycle simulator are used worldwide for wide range of applications:
+optimisation of the industrial operation of a existing nuclear fleet, assessing
+the future of nuclear energy and providing valuable informations required for
+the political decision, developement of tools dedicated to non-proliferation
+treaties verification. Moreover, those tools are used for Research and
+Development training as access point to key datas related to the fuel cycle. 
 
-The different institutions that use fuel cycle simulations pursue different
-goals. Consequently, their computational tools vary in term of complexity. The
-level of complexity of each software has to be coherent in regards to the
-different studies it is used for. To improve confidence in the results,
-institutions are tempted to increase the complexity of their software even if
-this complexity might not be necessary.
+The different institutions that use (and develop) fuel cycle simulators pursue
+different goals. Consequently, their computational needs vary in term of
+complexity and the level of complexity of each software pieces has to be in
+adequation with regards to their different simulations requirements. To improve
+confidence in the results, institutions are tempted to increase the complexity
+of their software even if this complexity might not be necessary.
 
 As an example, the neutron and gamma doses calculation requires the precise
 knowledge of each material isotopic composition in each facilities whereas
-uranium consumption calculation does not require this degree of detail. As a
+uranium consumption calculation does not require the same degree of detail. As a
 consequence, some software functionality may not be necessary regarding the
-technical question the code assesses. In other words, to solve a given technical
-question associated to a targeted precision, some functionality are needed while
-some others are not. Knowing the importance of each ones helps to choose an
-appropriate software or to engage code developments to solve the question. Also,
-some technical issues are assessed by numerous studies performed with different
-software and it is often difficult to compare them. Knowing the impact of
-functionality on different simulation outputs helps to estimate the confidence
-level of studies.
+technical question the code assesses: solving a given technical question
+associated with a targeted precision will required a limited set of
+functionality complexity. Knowing the importance of each ones helps to choose an
+appropriate software or to engage code developments to solve a specific
+question. Also, some technical issues are assessed by numerous studies performed
+with different software and it is often difficult to compare them. Knowing the
+impact of functionality on different simulation outputs helps to rank them by
```

Comment:
replace "rank" by a good way to write "to estimate the confidence level of studies.".

Reply:
OK pour la proposition de Nico